components/feature/sitepermissions/src/main/res/values/strings.xml

```diff
@@ -38,8 +38,11 @@
     access to this permission-->
     <string name="mozac_feature_sitepermissions_not_allow">Don\'t allow</string>
 
-    <!-- Accessibility text for indicating the permission icon in a permission request dialog -->
-    <string name="mozac_feature_sitepermissions_accessibility_permission_icon">Permission Icon</string>
+    <!-- Accessibility text for indicating the permission notification icon in a permission request dialog -->
+    <string name="mozac_feature_sitepermissions_accessibility_permission_location_icon">Location Icon</string>
+
+    <!-- Accessibility text for indicating the permission notification icon in a permission request dialog -->
+    <string name="mozac_feature_sitepermissions_accessibility_permission_notification_icon">Notification Icon</string>
```

Comment:
I think those content descriptions serve no purpose. Note that the ImageViews are set to ` android:importantForAccessibility="no"`. So they shouldn't require a content description.

Reply:
Good catch I didn't notice it 👍